src/components/LineChart/LineChart-container.tsx

```diff
@@ -10,7 +10,7 @@ import {
 import testData from './data';
 
 const LineCharts = (props: any): React.ReactElement => {
```

Comment:
dodaj typ :)

Reply:
This will be typed in next task when I know what type of data will be stored.